students/14th/team_6/hyun/project_westagram/user/views.py

```diff
@@ -0,0 +1,76 @@
+from django.views import View
+from .models import User
+from django.http import JsonResponse
+import json
+
+class SignUpView(View):
+    def post(self,request):
+        signup_data = json.loads(request.body)
+        try:
+
+            if User.objects.all().filter(
+                user_name = signup_data["name"]).exists():
+                return JsonResponse(
+                { 'message: ' : 'ID already exists'}, status = 400)
```

Comment:
```suggestion
            if User.objects.filter(
                user_name = signup_data["name"]).exists():
                return JsonResponse(
                { 'message: ' : 'ID already exists'}, status = 400)
```
`all()`로 모든 객체를 다 가져오지 않아도 `filter()` 메소드로 바로 조건별로 필터링을 하실 수 있습니다 :) 아래 에 있는 다른 부분에서도 필터링을 하실 것이라면 `all()` 을 꼭 쓰지 않으셔도 됩니다